src/main/java/com/djrapitops/extension/GriefDefenderExtension.java

```diff
@@ -81,6 +85,32 @@ public long claimCount(UUID playerUUID) {
         return getClaimsOf(playerUUID).size();
     }
 
+    @StringProvider(
+            text = "Claim Type",
+            description = "Basic + Town + SUBDIVISION + Admin",
+            iconName = "map-marker",
+            iconColor = Color.BLUE,
+            showInPlayerTable = true
+    )
+    public String claimType(UUID playerUUID) {
+        if (getClaimsOf(playerUUID) != null) {
+            getClaimsOf(playerUUID)
+                    .forEach(this::countType);
+        }
+        return b + "+" + t + "+" + s + "+" + a ;
```

Comment:
Consider creating a separate NumberProvider for each variable - Relying on the description to explain what the numbers mean can be confusing to users